packages/node-opcua-server/source/server_capabilities.ts

```diff
@@ -64,7 +64,7 @@ export interface IServerCapabilities {
     maxArrayLength: number;
     maxByteStringLength: number;
     maxQueryContinuationPoints: number;
-    minSupportedSampleRate: number;
+    minSupportedSampleRate: Double;
```

Comment:
isn't it Duration?

Reply:
Duration = Double = number in javascript